src/client/utils.js

```diff
@@ -49,6 +49,7 @@ utils.units = {
     if(pq.code == "[in_i]") return 2.54*pq.value;
     if(pq.code == "ft") return 30.48*pq.value;
     if(pq.code == "[ft_us]") return 30.48*pq.value;
+    if(pq.code == "[ft_i]") return 30.48*pq.value;
```

Comment:
Are we sure that the formula for ```ft``` , ```ft_us``` and ```ft_i``` is the same ?

Reply:
My assumption was based on this [comment](https://jira3.cerner.com/browse/OPENSUP-2094?focusedCommentId=8995974&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-8995974).

Reply:
Checked with a Clinical Terminologist in Cerner for the above. The calculations will be the same since ```ft``` , ```ft_us``` and ```ft_i``` all represent the same unit.